packages/frontend/src/commons/chains/form.ts

```diff
@@ -0,0 +1,62 @@
+import { SupportedDex } from "@metrom-xyz/sdk";
+import { SupportedChain, ADDRESS } from "@metrom-xyz/contracts";
+import { type ChainData } from "..";
+import { defineChain } from "viem";
+import { FormLogo } from "@/src/assets/logos/chains/form";
+import { FibonacciLogo } from "@/src/assets/logos/dexes/fibonacci";
+
+export const form = defineChain({
+    id: 478,
+    name: "Form",
+    nativeCurrency: {
+        decimals: 18,
+        name: "Ether",
+        symbol: "ETH",
+    },
+    rpcUrls: {
+        default: {
+            http: ["https://rpc.form.network/http"],
+        },
+    },
+    blockExplorers: {
+        default: {
+            name: "Blockscout",
+            url: "https://explorer.form.network/",
+        },
+    },
+    contracts: {
+        multicall3: {
+            address: "0xcA11bde05977b3631167028862bE2a173976CA11",
+            blockCreated: 1_214_281,
+        },
+    },
+});
+
+export const formData: ChainData = {
+    name: form.name,
+    metromContract: ADDRESS[SupportedChain.Form],
+    blockExplorers: form.blockExplorers,
+    icon: FormLogo,
+    dexes: [
+        {
+            slug: SupportedDex.Fibonacci,
+            logo: FibonacciLogo,
+            name: "Fibonacci",
+            addLiquidityUrl: "https://www.fibonacci-dex.xyz/pool/{target_pool}",
```

Comment:
I think it's `/pools` here.

Reply:
Addressed in 899f7a7154608f1645dc5165a5d9c170c9547449